scripts/setup_integration_tests.sh

```diff
@@ -11,7 +11,7 @@ set -o pipefail
 REPO=$(dirname $0)/../
 export OLLAMA_MODELS=${REPO}/test_data/models
 REGISTRY_SCHEME=https
-REGISTRY=registry.ollama.ai
+REGISTRY=ollama.com
```

Comment:
should we keep this on the subdomain? Just nervous about future implications here